src/nodes.coffee

```diff
@@ -1034,27 +1034,11 @@ exports.IdentifierLiteral = class IdentifierLiteral extends Literal
     return
       name: @value
 
-exports.CSXTag = class CSXTag extends IdentifierLiteral
```

Comment:
Just had to move this class to below `JSXIdentifier` so that I could inherit from it (specifically to inherit its `astType: -> 'JSXIdentifier'`)